implementations/rust/ockam/ockam_command/src/tcp/outlet/create.rs

```diff
@@ -79,19 +83,17 @@ impl CreateCommand {
     pub async fn async_run(self, ctx: &Context, opts: CommandGlobalOpts) -> miette::Result<()> {
         initialize_default_node(ctx, &opts).await?;
 
-        let node_name = opts.state.get_node_or_default(&self.at).await?.name();
```

Comment:
I like that it know happens on the target node and not on the node running the command